libglvnd.yaml

```
@@ -0,0 +1,120 @@
+package:
+  name: libglvnd
+  version: 1.7.0
+  epoch: 0
+  description: The GL Vendor-Neutral Dispatch library
+  copyright:
+    - license: GPL-2.0-or-later AND BSD-2-Clause AND BSD-3-Clause AND MIT AND Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gtk-doc
+      - intltool
+      - libtool
+      - libx11-dev
+      - libxext-dev
+      - libxml2-py3
+      - m4
+      - meson
+      - pkgconf
+      - pkgconf-dev
+      - python3
+      - xorgproto
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 2b6e15b06aafb4c0b6e2348124808cbd9b291c647299eaaba2e3202f51ff2f3d
+      uri: https://gitlab.freedesktop.org/glvnd/libglvnd/-/archive/v${{package.version}}/libglvnd-v${{package.version}}.tar.gz
+
+  - runs: |
+      ./autogen.sh
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    description: ${{package.name}} dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - libglvnd
+
+update:
+  enabled: true
+  release-monitor:
+    identifier: 12098
+
+test:
+  environment:
+    contents:
+      packages:
+        - build-base
+        - gcc
```

```suggestion
        - gcc
        - glibc-dev
```

If your recent commit doesn't fix the test, try adding this?

---

The tests seem to pass with this commit 🥳